.buildkite/run-tpu-v1-test.sh

```diff
@@ -19,16 +19,17 @@ docker run --privileged --net host --shm-size=16G -it \
     vllm-tpu /bin/bash -c "python3 -m pip install git+https://github.com/thuml/depyf.git \
     && python3 -m pip install pytest \
     && python3 -m pip install lm_eval[api]==0.4.4 \
+    && VLLM_USE_V1=1 VLLM_XLA_CHECK_RECOMPILATION=1
     && echo TEST_1 \
-    && VLLM_USE_V1=1 python3 /workspace/vllm/tests/tpu/test_compilation.py \
+    && python3 /workspace/vllm/tests/tpu/test_compilation.py \
     && echo TEST_2 \
-    && VLLM_USE_V1=1 pytest -v -s /workspace/vllm/tests/v1/tpu/test_basic.py \
+    && pytest -v -s /workspace/vllm/tests/v1/tpu/test_basic.py \
```

Comment:
in theory we should be able to run any new unit test we add to
 ```suggestion
    && pytest -v -s /workspace/vllm/tests/v1/tpu \
```
right?

Reply:
This is a good suggestion, thanks @NickLucche!

Reply:
Wait, not all tests under `tests/v1/tpu` are passing, like this one https://github.com/vllm-project/vllm/blob/main/.buildkite/run-tpu-v1-test.sh#L35. So if we pytest the folder it would fail.

A proper way to handle this I think it's to skip the failed test in python instead of commenting out in the shell script, I can do that in a separate PR.